docs/competition/volunteers/refreshments.md

```diff
@@ -0,0 +1,28 @@
+# Refreshments
+
+It's important that volunteers stay hydrated, which can be difficult for those in roles which don't allow them to move. (Don't give refreshments to competitors).
+
+**In SR2019, the refreshments are in the Volunteer Rest room, which is past the toilets and to the left. Also Water points are on both level 2 and 3 in the corner furthest from the toilets.**
```

Comment:
```suggestion
**In SR2019, the refreshments are in the Volunteer Rest room, which is past the toilets and to the left. Also water points are on both level 2 and 3 in the corner furthest from the toilets.**
```

Reply:
I've been learning too much German it seems